public/src/app/ressources/categorie/[slug]/page/[id]/page.tsx

```
@@ -86,13 +87,13 @@ export default async function RessourceCategoriePage({ params }: { params: { slu
   }
   const categories =  await fetchAPI('/categories',catQuery);
   const nbPage = meta ? meta.pagination.pageCount : 1;
-  const pageTitle= `Ressources de la catégorie ${categories.data.find((c:any) => c.attributes.slug = params.slug).attributes.label} page ${params.id}`; 
+  const pageTitle= `Ressources de la catégorie ${categories.data.find((c:any) => c.attributes.slug = slug).attributes.label} page ${id}`; 
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)